saleor/graphql/discount/types/promotions.py

```diff
@@ -82,19 +88,34 @@ class PromotionRule(ModelObjectType[models.PromotionRule]):
             AuthorizationFilters.AUTHENTICATED_STAFF_USER,
         ],
     )
+    reward_value = PositiveDecimal(
```

Comment:
Shoiuldn't we add ADDED_IN + PREVIEW_FEATURE?

Reply:
Same for `type` and `predicate_type`

Reply:
Done